commands/cdn/distribution/create.go

```diff
@@ -0,0 +1,113 @@
+package distribution
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/ionos-cloud/ionosctl/v6/internal/constants"
+	"github.com/ionos-cloud/ionosctl/v6/pkg/pointer"
+	"github.com/ionos-cloud/ionosctl/v6/pkg/uuidgen"
+
+	cdn "github.com/ionos-cloud/sdk-go-cdn"
+
+	"github.com/ionos-cloud/ionosctl/v6/internal/client"
+	"github.com/spf13/viper"
+
+	"github.com/ionos-cloud/ionosctl/v6/internal/core"
+)
+
+func Create() *core.Command {
+	cmd := core.NewCommand(context.Background(), nil, core.CommandBuilder{
```

Comment:
Is there a reason why we not to use `NewCommandWithJsonProperties` ?

The user might not know exactly what json we expect here for `--routing-rules`.

NewCommandWithJsonProperties adds two flags `--json-properties` and `--json-properties-example`. The latter if set, it prints an example json (which for example could be piped to a .json file), and exits, so the user can use that as a sort of guide

Reply:
JsonProperties is kind of vague. Having the custom name is more unambiguous. We can have a routing-rules example if we want.

Reply:
sure